input/community/committeelist/membership.yaml

```
@@ -1,7 +1,7 @@
 Title: Membership
 Description: The committee is responsible for evalutating new member applications and discussing membership criteria.
-Chairperson: TBD
-Image: https://via.placeholder.com/150.png?text=TBD
-Meets: 1st and 3rd Tuesday of each month at 1:30pm UTC
+Chairperson: Bill Wagner
+Image: /assets/members/bill-wagner.jpg
+Meets: TBD
```

Right now, it's 3:00 pm UTC:

```suggestion
Meets: 1st and 3rd Wednesday of each month at 3:00pm UTC
```